pkg/testdata/models/apis/lambda/0000-00-00/generator.yaml

```diff
@@ -25,3 +25,6 @@ resources:
           in:
             - 1
             - 2
+  CodeSigningConfig:
+    tags:
+      ignore: true
```

Comment:
For my own edification:  why is this necessary here for `Lambda.CodeSigningConfig` but not for other applicable test cases like `MemoryDB.User`?

Reply:
`CodeSigningConfig` doesn't support tags in its create call - https://docs.aws.amazon.com/sdk-for-go/api/service/lambda/#CreateCodeSigningConfigInput - and for the purposes of these tests we aren't adding the custom field to support it